src/rules.jl

```diff
@@ -141,6 +141,10 @@ end
     :(  SpecialFunctions.airybiprime($x)  )
 @define_diffrule SpecialFunctions.airybiprime(x) =
     :(  $x * SpecialFunctions.airybi($x)  )
+@define_diffrule SpecialFunctions.airyaix(x)     =
+    :(  sqrt($x) * SpecialFunctions.airyaix($x) + SpecialFunctions.airyaiprimex($x)  )
```

Comment:
I suggest using
```suggestion
    :(  SpecialFunctions.airyaiprimex($x) + sqrt($x) * SpecialFunctions.airyaix($x)  )
```
for consistency with the rule in SpecialFunctions: https://github.com/JuliaMath/SpecialFunctions.jl/blob/1febdd05902f5b3a3adc9915aba40b776aa3ea24/src/chainrules.jl#L19